internal/pkg/deploy/cloudformation/stack/workload.go

```diff
@@ -70,11 +70,12 @@ const (
 // RuntimeConfig represents configuration that's defined outside of the manifest file
 // that is needed to create a CloudFormation stack.
 type RuntimeConfig struct {
-	Image              *ECRImage         // Optional. Image location in an ECR repository.
-	AddonsTemplateURL  string            // Optional. S3 object URL for the addons template.
-	EnvFileARN         string            // Optional. S3 object ARN for the env file.
-	AdditionalTags     map[string]string // AdditionalTags are labels applied to resources in the workload stack.
-	CustomResourcesURL map[string]string // Mapping of Custom Resource Function Name to the S3 URL where the function zip file is stored.
+	Image               *ECRImage // Optional. Image location in an ECR repository.
+	AddonsTemplateURL   string    // Optional. S3 object URL for the addons template.
+	EnvFileARN          string    // Optional. S3 object ARN for the env file.
+	PermissionsBoundary string    // Optional. Name of the IAM Managed Policy to set a permissions boundary.
```

Comment:
same here, instead of putting `PermissionsBoundary` into the `RuntimeConfig` it'd fit better into the individual constructor configs since it's not a field that's generated from the execution of the command